src/types.h

```diff
@@ -597,6 +597,7 @@ const std::vector<ProbeItem> PROBE_LIST = {
     .show_in_userspace_list = true },
   { .name = "BEGIN", .aliases = { "BEGIN" }, .type = ProbeType::special },
   { .name = "END", .aliases = { "END" }, .type = ProbeType::special },
+  { .name = "bpftrace", .aliases = { "bpftrace" }, .type = ProbeType::special },
```

Comment:
Shall we make a `bt` alias? It'd be unified with the common extension of bpfscript files.